android/build.sh

```diff
@@ -20,8 +20,34 @@ THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BA
 #
 SHLVL=0
 
+# Clean build artifacts: keyman-engine.aar libaries, output and upload directories
+clean ( ) {
+  cd "$KEYMAN_ROOT/android"
+
+  find . -name "keyman-engine.aar" | while read fname; do
+    echo "Cleaning $fname"
+    rm $fname
+  done
+
+  if [ -d "$KEYMAN_ROOT/android/KMAPro/kMAPro/build/outputs" ]; then
+    echo "Cleaning KMAPro build outputs directory"
+    rm -rf "$KEYMAN_ROOT/android/KMAPro/kMAPro/build/outputs"
+  fi
+
+  if [ -d "$KEYMAN_ROOT/android/upload" ]; then
+    echo "Cleaning upload directory"
+    rm -rf "$KEYMAN_ROOT/android/upload"
+  fi
+}
```

Comment:
I just noticed the build steps for Android: "Test-14.0: Samples and Test projects" use KMEA/build.sh instead of this build.sh. I wonder if either:

1. I should move cleaning `keyman-engine.aar` to KMEA/build.sh or

2. Have `build.sh -debug` only clean and then exit. (keep it a separate build step)

Reply:
I'm opting for 2) exit after clean (l. 47)